Cargo.lock

```diff
@@ -1587,7 +1660,7 @@ dependencies = [
  "blake2b_simd",
  "byteorder",
  "digest 0.9.0",
- "jubjub",
+ "jubjub 0.9.0",
```

Comment:
This is what is keeping the older crate versions around.

Reply:
This is coming in through `zcash_proofs` via `redjubjub`.

Reply:
Fixed by upgrading to the redjubjub patch dependency post https://github.com/ZcashFoundation/redjubjub/pull/158. Depends on https://github.com/zcash/librustzcash/pull/801.